test/live/live.test.ts

```diff
@@ -183,6 +183,44 @@ describe('goerli', function () {
 			9000
 		)
 	}, 60000)
+
+	it.only('should create a native link and claim it (without pasing in tokenType', async function () {
```

Comment:
nice!

The live.test.ts file has pretty low code quality right now. I'd suggest for new tests added, try to avoid stuff like test-specific names if not necessary (e.g. goerliWallet  or goerliProvider when wallet would be sufficient)